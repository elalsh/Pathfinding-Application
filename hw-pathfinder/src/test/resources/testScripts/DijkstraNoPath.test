# Find a path with Dijkstra's algorithm when
# no path in the graph.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A B 1.0
AddEdge simpleGraph C B 9.0

FindPath simpleGraph B C