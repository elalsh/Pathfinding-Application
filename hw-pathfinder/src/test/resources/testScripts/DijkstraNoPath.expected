# Find a path with Dijkstra's algorithm when
# no path in the graph.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to B in simpleGraph
added edge 9.000 from C to B in simpleGraph

path from B to C:
no path found